apps/pushnotif-func/utils/notificationhubServicePartition.ts

```diff
@@ -67,3 +68,59 @@ export const buildNHClient = ({
   AZURE_NH_HUB_NAME,
 }: NotificationHubConfig): NotificationHubsClient =>
   new NotificationHubsClient(AZURE_NH_ENDPOINT, AZURE_NH_HUB_NAME);
+
+export class NotificationHubPartitionFactory {
+  #notificationHubPartitions: DisjoitedNotificationHubPartitionArray;
+  #partitionFour: NotificationHubsClient;
+  #partitionOne: NotificationHubsClient;
+  #partitionThree: NotificationHubsClient;
+  #partitionTwo: NotificationHubsClient;
```

Comment:
ugh

can't you use a map instead of four different variables?

Reply:
I opted for a `Record` cause with a Map we have to deal with the `undefined` case when using the `get` method, please take a look [d0521ed](https://github.com/pagopa/io-messages/pull/265/commits/d0521eda9c4dda6d888ab59afc758b22e3ff2714)